ci/sdk_python.go

```diff
@@ -19,7 +19,7 @@ const (
 )
 
 var (
-	pythonVersions = []string{"3.10", "3.11", "3.12"}
+	pythonVersions = []string{"3.10", "3.11"}
```

Comment:
:eyes:

Reply:
I added this in a first commit here, but then split in another PR. After rebasing looks like the git thinks I want to remove it!